src/modules/leaderboard.js

```diff
@@ -0,0 +1,51 @@
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const gameId = 'eE5VQjNtf7MXYoQKF5qi';
+const addScoreURL = `${baseURL + gameId}/scores/`;
+
+// to craete the game ID
+
+const createGame = async () => {
+  await fetch(baseURL, {
+    method: 'POST',
+    body: JSON.stringify({
+      name: 'My Leaderboard game',
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((response) => response.json())
+    .then((json) => (json));
+};
+
+function scoreTable(scores) {
+  let output = '';
+  scores.forEach((user) => {
+    output += `<li>${user.user}: ${user.score}</li>`;
+  });
+  document.querySelector('.lead-items').innerHTML = output;
+}
```

Comment:
- [OPTIONAL] As it is a best practice, " `usage of arrow functions instead of the function keyword` ", please it will be nice to convert your `scoreTable  function`  to an arrow function.